### PR TITLE
first

### PR DESCRIPTION
**Pull request by kylebcarlson**
Required change for [https://giphypedia.atlassian.net/browse/CXP-934?atlOrigin=eyJpIjoiNTA5YjQyNmRlMTMyNDU1N2JmMzg0NmUyNWY4NGQxZDkiLCJwIjoiaiJ9](url)

### CODE REVIEW

File: packages/react-components/src/components/video/index.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ type Props = {
     height?: number
     volume?: number
     className?: string
+    copied?: boolean
```
Comment (giannif) [inline packages/react-components/src/components/video/index.tsx:42]: @kylebcarlson this new property `copied` doesn't have any code associated with it. It looks like you're adding a link button in the overlay, which you can do in the web app, no? Maybe in one of the overlay components [here](https://github.com/Giphy/web-app/blob/master/app/static/js/shared/components/video-inline-play/index.tsx#L104)?

Comment (kylebcarlson) [reply packages/react-components/src/components/video/index.tsx:42]: made some changes to this PR https://github.com/Giphy/web-app/pull/3387/files
added the `copied` property so this component expects it: `shared/components/video-player/sdk-video-with-ga.tsx`

Comment (giannif) [reply packages/react-components/src/components/video/index.tsx:42]: @kylebcarlson I think we should find a way to add it to the web app then, follow the pattern of `gridPosition` there which looks to be used in a similar way and is not part of the SDK

Comment (kylebcarlson) [reply packages/react-components/src/components/video/index.tsx:42]: ah yeah, damn i should have seen that.

**Pull request closed without merging**